Makefile.am

```diff
@@ -286,7 +286,8 @@ MANPAGES = \
 	docs/check-update.timer.4 \
 	docs/swupd-update.service.4 \
 	docs/swupd-update.timer.4 \
-	docs/update-triggers.target.4
+	docs/update-triggers.target.4 \
+	docs/swupd-add-pkg.4
```

Comment:
If you had this in alphabetical order it would be even less of a change (one line added vs one line removed and 2 added)